### PR TITLE
Mosaic scale fix

### PR DESCRIPTION
**Pull request by iandobbie**
This pulls the displayed scaling from the camera view panel rather than the current image min/max values. Adds a number of getCurrentScaling functions based on the exisiting getScaling functions.

### CODE REVIEW

**Comment by mickp** [comment]
I think #431 might have the same effect, without the additional code - it just saves the scaling values at the time they're first calculated, rather than trying to recalculate them (with potentially different data) each time - they should only get recalculated when either:
 * the user drags a histogram bracket;
 * the imageViewer tiles are (re)set (i.e. on first creation, and when the image data size changes)

**Comment by carandraug** [comment]
TLDR: this PR can be reduced to undoing 31fa98e4ce5b8f67930d276fa625af314e3d323e and doing so would reintroduce the bug it fixed.

---

This PR replaces all calls to `getCameraScaling` and `getScaling` with new `getCurrentCameraScaling` and `getCurrentScaling`. Changing the methods name back to what they were, and only changing their implementations, the changes are reduced to replacing the implementation of `ViewCanvas.getScaling` with:

    return(self.tiles[0][0].imageMin, self.tiles[0][0].imageMax)

But the comments in `ViewCanvas.getScaling` have:

```
        # Used to query image data for imageMin and imageMax, but could this
        # occasionally hit an indexing error on the first image when images
        # were arriving fast. Just do the simple arithmetic here.
        #return (self.tiles[0][0].imageMin, self.tiles[0][0].imageMax)
```

which was written by @mickp in 31fa98e4ce5b8f67930d276fa625af314e3d323e

**Comment by mickp** [comment]
This is fixed by refactoring of the camera views in #447 , which should be merged soon.
